### PR TITLE
add support for up --scale

### PR DESCRIPTION
**Pull request by ndeloof**
**What I did**
introduced support for --scale

### CODE REVIEW

File: cli/cmd/compose/up.go [hunk 1]
```diff
@@ -201,6 +215,38 @@ func runCreateStart(ctx context.Context, opts upOptions, services []string) erro
 	return err
 }
 
+func applyScale(opts []string, project *types.Project) error {
```
Comment (gtardif) [inline cli/cmd/compose/up.go:218]: unit test on this one ?

File: cli/cmd/compose/up.go [hunk 2]
```diff
@@ -201,6 +215,38 @@ func runCreateStart(ctx context.Context, opts upOptions, services []string) erro
...
+				if err != nil {
+					return err
+				}
+				if service.Deploy == nil {
+					service.Deploy = &types.DeployConfig{}
+				}
+				count := uint64(replicas)
+				service.Deploy.Replicas = &count
+				project.Services[i] = service
+				continue SCALE
```
Comment (gtardif) [inline cli/cmd/compose/up.go:242]: errf... I will need to spend some time to look for a nicer solution, but ok (with some tests around it for now)

**Review by gtardif — COMMENTED** [review]
LGTM with unit tests on applyScale()

File: cli/cmd/compose/up.go (hunk 2 above)
Comment (ndeloof) [reply cli/cmd/compose/up.go:242]: I like using `for` labels for such scenarios. The other ways are either define sub-functions that would hide this `goto` by doing `return` inside the loop, or include a `var found bool` as a guard.

**Comment by gtardif** [comment]
https://github.com/docker/compose-cli/pull/1287/checks?check_run_id=1886378763#step:9:84 should be fixed with a rebase

**Review by gtardif — APPROVED** [review]

**Pull request merged**